### PR TITLE
Collabora: hide loading delay warning if document is loaded

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
- open document
- have a slow connection -> see loading snackbar
- wait for finish loading document
- previously: snackbar remained
- now: it is dismissed

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

File: src/main/java/com/owncloud/android/ui/asynctasks/LoadUrlTask.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
+/* Nextcloud Android Library is available under MIT license
```
Comment (AndyScherzinger) [inline src/main/java/com/owncloud/android/ui/asynctasks/LoadUrlTask.java]: wrong license, the app is GPL, the lib is MIT

**Comment by AndyScherzinger** [comment]
Looking good, just a minor license header comment

File: src/main/java/com/owncloud/android/ui/asynctasks/LoadUrlTask.java (hunk 1 above)
Comment (tobiasKaminsky) [reply src/main/java/com/owncloud/android/ui/asynctasks/LoadUrlTask.java]: Damn. Stupid copy&paste error.
Thanks :-)

**Pull request merged**